Test.sol

```diff
@@ -11,7 +11,17 @@ contract Test {
 		tlock = 0;
 	}
 
-	function requiresLock() external reentrancyGuard {
-		require(true, 'Yay');
+	function requiresLock() external reentrancyGuard returns (uint256) {
+		require(tlock == 1, 'Yay');
+        return 0xffff;
 	}
+
+    function bar() external pure returns (uint256) {
```

Comment:
IMO we should leave this contract as is—it's useful for `Test.sol`, `TestStorage.sol` and `TestYul.sol` to be simple and as similar as possible so we can diff the bytecode generated for each. Perhaps we showcase this in a different file?

Reply:
> IMO we should leave this contract as is—it's useful for `Test.sol`, `TestStorage.sol` and `TestYul.sol` to be simple and as similar as possible so we can diff the bytecode generated for each. Perhaps we showcase this in a different file?

That makes sense, I reverted the change in this file. Sorry about that